pkgs/tools/security/maigret/default.nix

```diff
@@ -82,6 +79,8 @@ python3.pkgs.buildPythonApplication rec {
     "test_pdf_report"
     "test_self_check_db_negative_enabled"
     "test_self_check_db_positive_enable"
+  ] ++ lib.optionals stdenv.isDarwin [
+    "test_asyncio_progressbar_executor"
```

Comment:
Please add a short comment here why the test is disabled

Reply:
Done, you are right it definitely needed a comment!